pytest_django/plugin.py

```diff
@@ -378,10 +378,10 @@ def _django_db_marker(request):
     This will dynamically request the ``db`` or ``transactional_db``
     fixtures as required by the django_db marker.
     """
-    marker = request.keywords.get('django_db', None)
+    marker = request.node.get_closest_marker('django_db')
     if marker:
-        validate_django_db(marker)
```

Comment:
I'm not 100% happy with the naming, but I didn't think of anything better.

Reply:
I was thinking `validate_django_db(…)` as it gets a value, and may happen to blow up, but it feels more ok to me now